examples/mip-tap/main.c

```diff
@@ -78,6 +99,10 @@ int main(int argc, char *argv[]) {
   mip_init(&mgr, &mif);
   MG_INFO(("Init done, starting main loop"));
 
+  // Start infinite event loop
+  MG_INFO(("Mongoose version : v%s", MG_VERSION));
+  MG_INFO(("Listening on     : %s", s_listening_address));
+
   extern void device_dashboard_fn(struct mg_connection *, int, void *, void *);
   mg_http_listen(&mgr, "http://0.0.0.0:8000", device_dashboard_fn, &mgr);
```

Comment:
`mg_http_listen(&mgr, s_listening_address, device_dashboard_fn, &mgr);`

Reply:
Failed ;-( thank you

Reply:
Just "humans being"